### ISSUE TITLE
What does the antenna look like?

### ISSUE THREAD

**Issue by romaindegivry** [AlexMontgomerie/4NEC2_helical_gen#5]
So we've got a definition for *n* concentric helices, But afterwards, I'm a bit stuck.

@AlexMontgomerie Can you figure out what geometry we want for the top connection? Also what type of connection do we want for the bottom edges (I assume one grounded and a voltage source on the other.

Check the file necpp_main.py for details

**Comment by AlexMontgomerie** [comment]
so for now, i think we can just short the end if the antenna is long enough (http://www.jhuapl.edu/techdigest/views/pdfs/V12_N1_1991/V12_N1_1991_Stilwell.pdf). And i think since this is a simulation, leaving them open won't really have much difference either. In the actual antenna we will need to balance the current along the 2 wires as there will be some mismatch, but i think the end isn't that much of an issue. 

In terms of driving it, we drive each wire with equal magnitude but opposite sign. 

So for our antenna, we will just need to gen 2 helix and rotate one by 180 degrees.

**Comment by AlexMontgomerie** [comment]
also, we don't need a ground plane for the antenna, but it might be interesting to place one later, to see what the effect of the satellite will have on the antenna

**Comment by romaindegivry** [comment]
ok, sounds good, I'll add in the short at the top.
Regarding the simulation, I suggest we try to find out the radiation pattern at our favorite frequency (406Hz I guess) , and run an optimization function (grad descent/GA probably) to figure out the best parameters (pitch, etc...)
Tell me what you think this way we can split the tasks, I'd like to have the code running monday if possible.

**Comment by AlexMontgomerie** [comment]
i reckon for splitting up tasks,

* finish geometry design and excitation feed 
* work on output interpretation for our application (getting radiation across a plane a certain distance away)
* parameter optimisation method (so input and feedback from the output) 

I'm happy to work on the output, if you want to finish creating the geometry. I'll get henry doing some stuff as well.

**Comment by henry-eshbaugh** [comment]
take great care placing the ground plane around the antenna, a solid sheet of conductor will interfere with the radiation pattern. See [this pic](http://colinkarpfinger.com/blog/wp-content/uploads/2010/05/IMG_1324-Custom.jpg); notice the keep-out under the business end.

From Stillwell 91: "A bifilar helix, on the other hand, is excited with equal and opposite currents on the two wires, so no ground plane is involved" -- I'd recommend a big keep-out.

**Comment by AlexMontgomerie** [comment]
yeah, i agree we should not put a ground plane for now, but wouldn't a ground plane be an easy way of simulating the effect of the satellite body on the antenna? I think ground plane's are only important for single feed antenna and small pcb antenna (in terms of impedance), and won't have much effect on our antenna. But yeah, lets not have one for now

**Comment by henry-eshbaugh** [comment]
IIRC common wisdom is that any conductor in the path of your beam will interfere with the signal. antennas are voodoo, best to be safe. the antenna should ideally be a deployable.
